busted/outputHandlers/TAP.lua

```diff
@@ -31,12 +40,14 @@ return function(options)
       message = pretty.write(message)
     end
 
-    print(failure:format(counter, t.name))
+    print(string_format(failure, counter, t.name))
```

Comment:
I get what you are doing with al these function replacements, but as I understand the original issue report and PR, the `print()` function itself also needs to be cached, but I don't see that happening in this PR. Am I missing something?

Reply:
more broadly; the outputters should cache the functions they use, since I replaced all usage of `print`, it no longer needs to be cached, since the same applies to other functions it does cache `format`, `gsub`, etc.

Reply:
Neverbmind, I mussed L9 entirely! That answers my concern.